GPflow/gpflowrc

```diff
@@ -9,6 +9,8 @@ int_type = int32
 
 [numerics]
 jitter_level = 1e-6
+# quadrature can be set to: allow, warn, error
+quadrature = warn
```

Comment:
Should this be `ekern_quadrature`, instead of just `quadrature`? We use quadrature as standard for many likelihoods.